### DISCUSSION TITLE
Add markdown-oxide to macos (homebrew)

### DISCUSSION THREAD

**Discussion by Feel-ix-343** [Feel-ix-343/markdown-oxide#23] [Ideas]
Add markdown-oxide to homebrew. This is related to adding markdown-oxide support to Zed.dev  in this PR https://github.com/zed-industries/zed/pull/8463

**Comment by Feel-ix-343** [comment]
@sQVe do you have any experience publishing to homebrew? I both do not and do not use macos.

**Comment by sQVe** [comment]
@Feel-ix-343 I do not. I've been an Arch Linux user for the past 10 years, and have little experience with the MacOS ecosystem. There should likely be a guide in their repo though.

**Comment by sQVe** [comment]
I would assume that you would need to distribute releases here on GitHub, if you would want to have things on Homebrew. I would be surprised if it allows for downloading Rust dependencies, building, and etc.

**Comment by sQVe** [comment]
If the goal is to make it easier to install `markdown-oxide`, the first step I would take is to provide releases here on GitHub, since you basically just need the binary.

**Comment by Feel-ix-343** [comment]
Thanks for the tips 🙏 Ill get it done. 

Note Ill probably provide a nightly binary
